forward_system/src/run/output.rs

```diff
@@ -134,12 +135,21 @@ impl<TR: TxResultCallback> From<ForwardRunningResultKeeper<TR>> for BlockOutput
             storage_writes,
             account_diffs,
             published_preimages,
-            pubdata,
+            // TODO: should we define 2 types of output?
```

Comment:
I would remove pubdata from block output, and for now return pubdata as separate value(not in block output). In future we'll probably have `BatchOutput`

Reply:
Removed it, had to use a new branch for interface